qa/setup_packages.py

```diff
@@ -512,7 +514,8 @@ def print_configs(cuda_version):
         for v in pckg.get_all_versions(cuda_version):
             alias = BasePackage.get_alias(v)
             if alias is not None:
-                v = "{}=={}".format(alias, v)
+                op = "" if str(v)[0] in ("<", ">", "=") else "=="
```

Comment:
nitpick, you can go with 
```suggestion
                op = "" if str(v)[0] in "<>=") else "=="
```

Reply:
I know, it was a deliberate choice.